examples/GainPlugin/jsui/src/App.js

```diff
@@ -51,13 +54,24 @@ class App extends Component {
     const muteBackgroundColor = this.state.muted ? 'ff66FDCF' : 'ff17191f';
     const muteTextColor = this.state.muted ? 'ff17191f' : 'ff66FDCF';
 
+    const sliderColor = 'ff66FDCF';
+
+    //TODO: Add  Label to Slider.
+    //   <Label paramId="MainGain" {...styles.label} />
+
     return (
       <View {...styles.container}>
         <View {...styles.content}>
           <Image source={require('./logo.png')} {...styles.logo} />
-          <Slider paramId="MainGain" {...styles.knob}>
-            <Label paramId="MainGain" {...styles.label} />
-          </Slider>
+          <ParameterSlider
+            paramId="MainGain"
+            sliderStrokeColor={sliderColor}
```

Comment:
Looks like `ParameterSlider` doesn't need these stroke/fill props, since that info is captured in the `Slider.drawLinearVertical` closure